Shared/mods/deathmatch/logic/CLodModels.cpp

```diff
@@ -0,0 +1,4388 @@
+/*****************************************************************************
+ *
+ *  PROJECT:     Multi Theft Auto
+ *               (Shared logic for modifications)
+ *  LICENSE:     See LICENSE in the top level directory
+ *  FILE:        mods/shared_logic/CLodModels.cpp
+ *
+ *****************************************************************************/
+
+#include "StdInc.h"
+#include "CLodModels.h"
+
+constexpr std::size_t OBJ_LOD_MODELS_COUNT = 4289;
+constexpr std::pair<std::uint32_t, std::uint32_t> OBJ_LOD_MODELS_ARRAY[] = {
+    {694, 784},                // sm_redwoodgrp => lod_redwoodgrp (countryS)
```

Comment:
maybe too much spacing before the comment?
`{X, Y}, // comment`
instead of
`{X, Y},         // comment`